vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -182,17 +182,42 @@ void FlyoutShadowNode::createView() {
         }
       });
 
-  // Turn off AllowFocusOnInteraction so that focus cannot land in the top
-  // content element of the flyout. If focus lands in the flyout presenter, then
-  // on moving focus somewhere else, including the children on the flyout
-  // presenter, the flyout can dismiss.
+  // After opening the flyout, turn off AllowFocusOnInteraction so that
+  // focus cannot land in the top content element of the flyout. If focus
+  // lands in the flyout presenter, then on moving focus somewhere else,
+  // including the children on the flyout presenter, the flyout can dismiss.
+  // (Wait until after opening, so that the flyout presenter has a chance to
+  // move focus to the first focusable element in the flyout.)
   //
-  // However, turning AllowFocusOnInteraction off at the root of the flyout and
+  m_flyoutOpenedRevoker =
+      m_flyout.Opened(winrt::auto_revoke, [=](auto &&, auto &&) {
+        auto instance = wkinstance.lock();
+
+        m_flyout.AllowFocusOnInteraction(false);
+
+        if (!m_updating && instance != nullptr) {
+          if (auto flyoutPresenter = GetFlyoutPresenter()) {
+            // When multiple flyouts are overlapping, XAML's theme shadows
+            // don't render properly. As a workaround we enable a z-index
+            // translation based on an elevation derived from the count of
+            // open flyouts. We apply this translation on open of the flyout.
+            // (Translation is only supported on RS5+, eg. IUIElement9)
+            if (auto uiElement9 = GetView().try_as<winrt::IUIElement9>()) {
+              winrt::Numerics::float3 translation{
+                  0, 0, (float)16 * s_cOpenFlyouts};
+              flyoutPresenter.Translation(translation);
+            }
+
+            flyoutPresenter.AllowFocusOnInteraction(false);
```

Comment:
Why do we need to set this again in here?

Reply:
+1 to this question

---
In reply to: [328731391](https://github.com/microsoft/react-native-windows/pull/3192#discussion_r328731391) [](ancestors = 328731391)

Reply:
In the earlier fix, setting AllowFocusOnInteraction false on the flyout prior to opening caused the associated flyoutpresenter to inherit that setting.  With this fix, that doesn't happen so we need to explicitly set it on the flyoutpresenter after opening.

---
In reply to: [328734469](https://github.com/microsoft/react-native-windows/pull/3192#discussion_r328734469) [](ancestors = 328734469,328731391)